definitions/variable/energy/energy-secondary.yaml

```diff
@@ -56,6 +56,24 @@
 - Secondary Energy|Hydrogen|Fossil|w/o CCS:
     description: Hydrogen production from fossil fuels with freely vented CO2 emissions
     unit: EJ/yr
+- Secondary Energy|Hydrogen|Coal:
```

Comment:
Might as well use "Secondary Energy|Hydrogen|<Fuel>" and remove the per-fuel variables...

Reply:
I have to admit I don't fully understand the comment?
or do you mean I should simply commit "Secondary Energy|Hydrogen|<Fuel>" and remove the explicit types?

Reply:
yes, please re-insert `Secondary Energy|Hydrogen|<Fuel>` and delete all explicit hydrogen-fuel variables (double-checking that the fuel is included in the Fuel list).